src/taipy/rest/api/resources/datanode.py

```diff
@@ -48,6 +49,15 @@
 REPOSITORY = "data"
 
 
+def get_or_raise(data_node_id: str) -> None:
+    """Raise an exception if the data node does not exist."""
```

Comment:
Same here: this function MAY return something valid.
And it's probably a private function. (-> `_get(id: str) -> DataNode`)